server/services/approverCaseloadService.ts

```diff
@@ -97,38 +119,74 @@ export default class ApproverCaseloadService {
 
     const coms = await this.communityService.getStaffDetailsByUsernameList(comUsernames)
 
-    return caseload.map(offender => {
```

Comment:
slight formatting change here - which is a bit confusing but code remains the same